test/performance_tests/performance_tests.vcxproj

```diff
@@ -1,183 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please revert back this and unit tests VS project files. As I mentioned earlier not everybody has CMake so we keep these files for now until we make a full integration of CUDA and OpenCL.